roles/telemetry_autoscaling/tasks/configure_heat.yml

```diff
@@ -27,19 +27,19 @@
 
   # TODO: Pass the parameters correctly. When I tried to pass the parameters
   # into the template.yaml file, they weren't passed to the instance initially.
-- name: RHOSO-12657 Configure heat template for automatically scaling instances
```

Comment:
For now, I think we need to keep the test IDs, since the custom logger is used to determine pass/fail, and it doesn't create a test_run_result.out file unless there are tests with test id prefix (i.e. RHOSO, RHOSP, UI)

Reply:
I'm going ot update this to match what is done in the other tests i.e. add a second line with the test id to the task name